src/server.js

```diff
@@ -365,6 +365,22 @@ function startBuilds(buildData) {
 	}));
 }
 
+/**
+ * Checks skip triggers and starts all builds needed for `prId` (which should be in the form 'User/Repo/123') as needed.
+ */
+function buildPullRequestIfNeeded(prId) {
```

Comment:
`buildPullRequest` returns a `ReplaySubject`; `buildPullRequestIfNeeded` returns a `Promise`; can those be used interchangeably?

Reply:
It looked like the return value of the places I changed to `buildPullRequestIfNeeded` weren't using the return value, so I *think* that shouldn't matter.